Makefile

```diff
@@ -102,6 +107,12 @@ $(BINDIR)/deepcopy-gen: .init cmd/libs/go2idl/deepcopy-gen
 	  -i $(SC_PKG)/pkg/apis/servicecatalog,$(SC_PKG)/pkg/apis/servicecatalog/v1alpha1 \
 	  --bounding-dirs github.com/kubernetes-incubator/service-catalog \
 	  -O zz_generated.deepcopy
+	$(DOCKER_CMD) $(BINDIR)/conversion-gen --v 5 --logtostderr \
+	  -i $(SC_PKG)/pkg/apis/servicecatalog,$(SC_PKG)/pkg/apis/servicecatalog/v1alpha1 \
+	  --extra-peer-dirs k8s.io/kubernetes/pkg/api,k8s.io/kubernetes/pkg/api/v1,k8s.io/kubernetes/pkg/apis/meta/v1,k8s.io/kubernetes/pkg/conversion,k8s.io/kubernetes/pkg/runtime \
+	  -O zz_generated.conversion
+	  # the previous three directories will be changed from kubernetes to apimachinery in the future
+	$(DOCKER_CMD) $(ROOT)hack/update-codecgen.sh
```

Comment:
Note to reviewers: I am totally happy to inline this into the makefile in a follow-up, but would prefer to put in as-is for now.